nsv13/code/modules/projectiles/projectile.dm

```diff
@@ -8,9 +8,6 @@ GLOBAL_LIST_INIT(projectile_hitbox, list(new /datum/vector2d(-2,16),\
 	var/datum/component/physics2d/physics2d = null
 	var/obj/structure/overmap/overmap_firer = null
 
-/obj/item/projectile/forceMove(atom/destination)
-	return doMove(destination)
-
```

Comment:
...I'm assuming you haven't tested overmap projectiles yet

Reply:
I've tested them plenty, if you can tell me exactly what this is supposed to fix I can probably implement that fix without screwing up the projectile's forcemove()

Reply:
Oh. Weird. If it works it works!